includes/attendee/attendee-repository.php

```diff
@@ -108,11 +108,38 @@ public function get_attendee( int $event_id, int $user_id ): ?Attendee {
 	}
 
 	/**
+	 * Get all attendees for an event, with or without contributions.
+	 *
+	 * @param int $event_id The id of the event.
+	 *
 	 * @return Attendee[] Attendees of the event.
 	 */
-	public function get_attendees( int $event_id ): array { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
-		// TODO.
-		return array();
+	public function get_attendees( int $event_id ): array {
+	 // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
+		global $wpdb, $gp_table_prefix;
+		$attendees = array();
+
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		$rows = $wpdb->get_results(
+			$wpdb->prepare(
+				"
+				select event_id, user_id
```

Comment:
I think we should add `is_host` here and then call `mark_as_host()` on the attendee if `'1' === $row->is_host`. Currently the method returns hosts attendees (as it should, I believe) but `is_host()` will return false even if the attendee is a host.

Reply:
I have updated this method from the `trunk` branch, with this update. Thank you.